__tests__/e2e/06_Label.test.js

```diff
@@ -0,0 +1,47 @@
+import test from 'tape-async'
```

Comment:
I think that we should rename file with e2e test (__tests__/e2e/06_Label.test.js), because we already have e2e test file with the same prefix "06_" (__tests__/e2e/06_LabelRating.test.js)